resident/resident-service/src/main/java/io/mosip/resident/service/impl/ProxyMasterdataServiceImpl.java

```diff
@@ -42,7 +71,28 @@ public class ProxyMasterdataServiceImpl implements ProxyMasterdataService {
 
 	@Autowired
 	Environment env;
+	
+	@Autowired
+	private ProxyMasterdataService proxyMasterdataService;
+	
+	private TemplateManager templateManager;
 
+	@Autowired
+	private TemplateManagerBuilder templateManagerBuilder;
+
+	@PostConstruct
+	public void idTemplateManagerPostConstruct() {
```

Comment:
why are we mixing variables and methods in a single place. All variables should be in the top and all member functions should be in the end.

Reply:
modified the code as per the review comments